### DISCUSSION TITLE
Question: [Can Mica be used in Web Apps running in Edge WebView]

### DISCUSSION THREAD

**Discussion by antonioberetini** [microsoft/microsoft-ui-xaml#7457] [Q&A]
We are working on a progressive web application which we plan on submitting to windows store. We are toying around using Mica to make our layout match what has been implemented in the new Windows Store on Windows 11.

**Comment by StephenLPeters** [comment]
@jevansaks or @codendone can you answer?

**Comment by antonioberetini** [comment]
Maybe better question would be if Edge WebView 2 supports Mica Material

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by antonioberetini** [comment]
Any update on this?

**Comment by bastecklein** [comment]
This would be a great feature for PWAs…
